database/db.go

```diff
@@ -0,0 +1,24 @@
+package database
+
+import (
+	"os"
+	"redtower/service/database/models"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
+
+func ConnectDatabase()  {
+	dsn := os.Getenv("DSN")
```

Comment:
Make sure this is how akshull envs work in Go, also add an `.env.example` and then make something in the `.README` file about this.

Reply:
Finished the first part of this will have to make example and README updates later